zerver/webhooks/taiga/view.py

```diff
@@ -23,217 +22,237 @@
 def api_taiga_webhook(request: HttpRequest, user_profile: UserProfile,
                       message: Dict[str, Any]=REQ(argument_type='body')) -> HttpResponse:
     parsed_events = parse_message(message)
-
     content_lines = []
     for event in parsed_events:
         content_lines.append(generate_content(event) + '\n')
-    content = "".join(sorted(content_lines))
+    content = ''.join(sorted(content_lines))
```

Comment:
Why did you change all the `""` to `''` in this file?  It makes your changes very hard to read, and is not in line with the conventions of the codebase.  Can you please undo all the irrelevant changes where all you did was changed the quote format?